VRCFaceTracking/OSC/OSCParams.cs

```diff
@@ -45,13 +45,17 @@ public BaseParam(string name, Type type)
             {
                 _paramName = name;
                 _paramType = type;
-                OscType = Utils.TypeConversions[type].oscType;
+                OscType = OscUtils.TypeConversions[type].oscType;
             }
 
             public virtual void ResetParam(ConfigParser.Parameter[] newParams)
             {
-                var compatibleParam =
-                    newParams.FirstOrDefault(param => param.name == _paramName && param.input.Type == _paramType);
+                Regex regex = new Regex(@"(?<!(v\d+))(/" + _paramName + @")$|^(" + _paramName + @")$");
+
+                var compatibleParam = newParams.FirstOrDefault(param =>
+                    regex.IsMatch(param.name)
+                    && param.input.Type == _paramType);
```

Comment:
I wonder if there might be something that could be improved with this regex pattern? It's meant to allow wildcard prefixes and avoids any baked in prefixes (eg. v2/-x-). Anyhow the performance seems fine and doesn't affect the parameter sending performance.

Reply:
I'd add a comment explaining the regex at a glance and the purpose of the two parameter naming conventions

Reply:
Also v1/foo/bar would match but v1/bar wouldn't. Is that intentional?

Reply:
Yes, though the format of VRCFT parameters will ensure that isn't really an issue (you would have to intentionally name a Unified parameter `foo/bar` to have that happen).

Reply:
I guess an extra `/` should invalidate the entire row